src/services/endpoint.ts

```diff
@@ -525,7 +525,7 @@ const hasEmptyCondition = <TEntity>(query: SelectQueryBuilder<TEntity>): boolean
   }
 
   // condition should contain at least one parameter or equal (number | string)
-  return !/([a-z\s."]+)([!=<>]+|like|is|in)\s?(:|\(:|[0-9]|')/i.test(condition);
+  return !/([a-z\s."]+)([!=<>]+|like|is|in)\s?(:|\(:| [0-9]|')/i.test(condition);
```

Comment:
you add space? use `\s`

Reply:
I am not the one who did it. Seems strange 
<img width="1013" alt="image" src="https://github.com/Lomray-Software/microservice-helpers/assets/44732463/892ae00c-e249-4492-8601-497049a49fe4">

Reply:
I'll revent "no space"